src/args.rs

```diff
@@ -87,6 +95,11 @@ pub fn print_usage() -> Result<Option<Config>, Error> {
     } else {
         ""
     };
+    let copy_path = if SUPPORTS_CLIPBOARD {
+        "\n    -p, --copy-path\n            Copy the local destination path to clipboard after cloning.\n"
```

Comment:
Please add this to the existing `if`, and `clipboard` variable instead of adding a separate one.